static_src/test/functional/user_invite.spec.js

```diff
@@ -31,18 +31,6 @@ describe('User roles', function () {
       expect(email).toBe(userInviteElement.getInviteFormValue());
     });
 
-    it('should be able to submit an email address and see on user list', function () {
```

Comment:
This whole thing doesn't need to be removed. I do want to disable the last expectation though, because that functionality is changing right away when we write the loading state code.